sdks/python/apache_beam/io/kafka.py

```diff
@@ -152,8 +155,18 @@ def __init__(
         for tests and demo applications.
     :param max_read_time: Maximum amount of time in seconds the transform
         executes. Mainly used for tests and demo applications.
+    :param commit_offset_in_finalize: Whether to commit offsets when finalizing.
+    :param timestamp_policy: The built-in timestamp policy which is used for
+        extracting timestamp from KafkaRecord.
     :param expansion_service: The address (host:port) of the ExpansionService.
     """
+    if timestamp_policy not in [ReadFromKafka.processing_time_policy,
+                                ReadFromKafka.create_time_policy,
+                                ReadFromKafka.log_append_time]:
+      raise ValueError(
+          'timestamp_policy should be one of '
+          '[ProcessingTime, CreateTime, LogAppendTime]')
```

Comment:
nit: you might assign the allowed policies to a variable and re-use it to generate this error message.